samples/plugins/postgres_odbc/connectionBuilder.js

```diff
@@ -12,6 +12,12 @@
     params["UseDeclareFetch"] = "1";
     params["Fetch"] = "2048";
 
+    if (attr[connectionHelper.attributeSSLMode] == "require")
+    {        
+        params["sslmode"] = "require";
```

Comment:
required attribute: `<attr>sslmode</attr>`

Reply:
Thanks for catching that, opened https://github.com/tableau/connector-plugin-sdk/pull/294